src/ImageSharp/Image{TPixel}.cs

```diff
@@ -22,6 +22,7 @@ public sealed class Image<TPixel> : IImage, IConfigurable
     {
         private readonly Configuration configuration;
         private readonly ImageFrameCollection<TPixel> frames;
+        private readonly TPixel clearColor;
```

Comment:
No need for field. Assign straight to the property. (When you move all this to `ImageFrame`)

Reply:
Done.